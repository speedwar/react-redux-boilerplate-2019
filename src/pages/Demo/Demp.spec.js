import React from 'react'
import { shallow } from 'enzyme'
import Demo from './Demo'

const setup = () => {
  const wrapper = shallow(<Demo />)
  return { wrapper }
}

describe('<Demo /> component', () => {
  it('renders component', () => {
    const wrapper = setup()
    expect(wrapper).toMatchSnapshot()
  })
})
