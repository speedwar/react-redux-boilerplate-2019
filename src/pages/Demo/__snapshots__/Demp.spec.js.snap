// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Demo /> component renders component 1`] = `
Object {
  "wrapper": <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <BrowserRouter>
      <Layout />
    </BrowserRouter>
  </Provider>,
}
`;
